packages/ibm-products/src/components/Datagrid/Datagrid.test.js

```diff
@@ -913,6 +942,21 @@ describe(componentName, () => {
     );
   });
 
+  it('renders a table with spacer column', () => {
+    render(<SpacerColumn />);
+    expect(screen.getByRole('table')).toHaveClass(
+      `${carbon.prefix}--data-table`
+    );
+
+    expect(
+      screen
```

Comment:
A better way to get all table headers could be to do something like this:
```
expect(screen.getAllByRole('columnheader').length).toEqual(3);
```

This way you don't have to chain multiple `getElementsByTagName`.